kernel-modules/support-packages/04-create-support-packages.sh

```diff
@@ -40,7 +47,7 @@ for mod_ver_dir in "${MD_DIR}/module-versions"/*; do
 
     package_out_dir="${OUT_DIR}/${mod_ver}"
     mkdir -p "$package_out_dir"
-    if [[ "${mod_ver}" =~ [0-9]+\.[0-9]+\.[0-9]+ ]]; then
+    if [[ "${mod_ver}" =~ [0-9]+\.[0-9]+\.[0-9]+(:?-rc[0-9])? ]]; then
```

Comment:
That's quite interesting, can you explain to me this shell magic, why the last part is matching more than one digit after `-rc`?

Reply:
This is leftover logic to truncate the version of the support packages to 6 characters when they were versioned with sha digests, if it has a semver like versioning, then we use it as is. I think we already dropped support for the last driver versioned with shas, so we can probably drop the check altogether.